### DISCUSSION TITLE
Error 500 when calling auth.api.resetPasswordForEmail

### DISCUSSION THREAD

**Discussion by nenostra** [supabase/supabase#2045] [Questions]
Hi 👋🏻.
I am trying to implement password reset for my website, but when I call `auth.api.resetPasswordForEmail`, I receive the following error: `{"code":500,"msg":"Error recovering user","error_id":"5b3f85e2-240c-43fd-878e-64a84068bc29"}`.
I receive the same error if I try to reset password from the dashboard.
Thanks beforehand for the help!

**Comment by awalias** [comment]
Hey @nenostra ! There is an auth server log in the dashboard which will give you some pointers, it's in Auth > Settings and scroll to the bottom

![image](https://user-images.githubusercontent.com/458736/122943048-4249d200-d3a9-11eb-86c4-5cd59a56ad3e.png)

**Reply by nenostra** [reply]
Thanks for quick reply @awalias , wasn't aware of this.
This is an example of the log I'm getting:
`time="2021-06-22T08:00:40Z" level=error msg="500: Error recovering user" component=api error="Error sending recovery email: 535 Authentication failed: The provided authorization grant is invalid, expired, or revoked" method=POST path=/recover referer= remote_addr="18.237.78.104:57428" request_id=f1567386-ef04-4880-a900-2bf715eabcbb
`.
 But still don't understand what this means, also strange that I cannot even send password resets from the auth dashboard. Any help appreciated.

**Reply by nenostra** [reply]
@awalias any help here appreciated, I don't know how to proceed and I cannot find anything related on the docs. I don't have any custom configurations or anything, I'm just calling `auth.api.resetPasswordForEmail`
